deepchem/models/torch_models/prot_bert.py

```diff
@@ -0,0 +1,108 @@
+from transformers import BertForMaskedLM, BertForSequenceClassification, BertTokenizer, BertConfig
+import torch.nn as nn
+from deepchem.models.torch_models.hf_models import HuggingFaceModel
+from typing import Union
+
+
+class ProtBERT(HuggingFaceModel):
+    """
+    ProtBERT model[1].
+
+    ProtBERT model is based on BERT architecture and the current implementation
```

Comment:
@Shiva-sankaran I think a larger description of the model (ProtBert) will be better.
If anyone using this model can just read it and understand how the model works internally will be very useful.

Reply:
Added.